tutorials-and-examples/inference-servers/jetstream/maxtext/single-host-inference/deployment.yaml

```diff
@@ -61,4 +67,89 @@ spec:
     name: jetstream-grpc
     port: 9000
     targetPort: 9000
-
+  - protocol: TCP
+    name: prometheus-http
+    port: 9091
+    targetPort: 9091
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: prometheus-svc
+spec:
+  selector:
+    app: prometheus-server
+  ports:
+  - protocol: TCP
+    name: prometheus-client-http
+    port: 9090
+    targetPort: 9090
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: prometheus-server-conf
+data:
+  prometheus.yml: |
+    global:
+      scrape_interval: 500ms
+      evaluation_interval: 500ms
+    scrape_configs:
+      - job_name: 'prometheus'
+        kubernetes_sd_configs:
+        - role: pod
+          selectors:
+            - role: pod
+              label: "app=maxengine-server"
+        relabel_configs:
+          - source_labels: [__address__]
+            action: replace
+            regex: ([^:]+):.*
+            replacement: $1:9091
+            target_label: __address__
```

Comment:
if we use "jetStream_prefill_queue_size_<uuid>", what would be the yaml spec here?